src/current/_includes/releases/v24.3/v24.3.14.md

```diff
@@ -0,0 +1,84 @@
+- Fixed a bug where manually updating the `show` or `status` parameters in the URL (e.g., `http://127.0.0.1:8080/#/schedules?status=ACTIVE&show=50`) caused the **Schedules** page to fail to load.
+ [#144805][#144805]
+- Fixed a bug in the **SQL Activity Statements** page where filtering by **Statement Type** returned no results. The filter now works as expected.
+ [#144854][#144854]
+- Improve the performance of `SHOW CREATE TABLE` on multi-region databases with large numbers of objects.
+ [#145073][#145073]
+- Fixed a bug where running `DROP INDEX` on a hash-sharded index did not properly detect dependencies from functions and procedures on the shard column. This caused the `DROP INDEX` statement to fail with an internal validation error. Now the statement returns a correct error message, and using `DROP INDEX ... CASCADE` works as expected by dropping the dependent functions and procedures.
+ [#145392][#145392]
+- Fixed a bug where a node that was drained as part of decommissioning may have interrupted SQL connections that were still active during drain (and for which drain would have been expected to wait).
+ [#145447][#145447]
+- Fixed a bug that could lead to schema changes hanging after a cluster recovered from availability issues.
+ [#145543][#145543]
+- Previously, on a table with multiple column families, CockroachDB could encounter a `Non-nullable column "‹×›:‹×›" with no value` error in rare cases during table statistics collection. The bug was present since v19.2 and is now fixed.
+ [#145574][#145574]
+- Fixed a bug that could cause a row-level TTL job to fail with the error "comparison of two different versions of enum" if an `ENUM` type referenced by the table experienced a schema change.
+ [#145915][#145915]
+- Fixed a bug where the physical cluster replication (PCR) reader catalog job could hit validation errors when schema objects had dependencies between them (for example, when a sequence's default expression was being removed).
+ [#145997][#145997]
+- Fixed a bug where orphaned leases were not properly cleaned up.
+ [#146096][#146096]
+- Fixed an internal assertion failure that could occur during operations like `ALTER TYPE` or `ALTER DATABASE ... ADD REGION` when temporary tables were present.
+ [#146198][#146198]
+- Fixed a bug that could cause queries that perform work in parallel to ignore the requested quality-of-service level. Affected operations include lookup joins, DistSQL execution, and foreign-key checks.
+ [#146222][#146222]
+- Fixed a bug that prevented `TRUNCATE` from succeeding if any indexes on the table had back-reference dependencies, such as from a view or function referencing the index.
+ [#146324][#146324]
+- Fixed a bug where an invalid comment in the `system.comment` table for a schema object could make it inacessible.
+ [#146416][#146416]
+
+<h3 id="v24-3-14-miscellaneous">Miscellaneous</h3>
```

Comment:
I would move the three notes under Misc to another more descriptive category. Probably bug fixes and operational changes.